pallets/loans/src/lib.rs

```diff
@@ -110,9 +110,6 @@ pub mod module {
         /// Weight information for extrinsics in this pallet.
         type WeightInfo: WeightInfo;
 
-        /// Block per year
-        type BlockPerYear: Get<u128>;
-
         /// Unix time
```

Comment:
@GopherJ Can we also remove https://github.com/parallel-finance/parallel/blob/master/pallets/loans/src/rate.rs#L44

Reply:
I'll address it after lunch